WildFly_17.0.1.Final/jakarta-web-profile.adoc

```diff
@@ -1,169 +0,0 @@
-= TCK Results
```

Comment:
Was deleting this file intentional?

Reply:
Yes, I wanted to start a new EE 10 branch that doesn't include results from EE 8.  We probably could make that change on the `master branch` (change to `main` also).

Reply:
oh, sure, of course!